vtex-apps/product-summary
# Set a specific SKU in the product context

#### What problem is this solving?

There was no way of specifying which SKU should be the initial selection in product summaries (Both sliders and galleries).
We are now letting the client specify a logic to then use it for the product-context, which in turn will provide the selected item property to it's childrens.

#### How to test it?

[CHEAPEST](https://cheapest--motoroladevelopment.myvtex.com/)
(You will see the cheapest products selected)

[SPECIFICATION](https://specification--motoroladevelopment.myvtex.com/Smartphones/Moto-G-Family?page=1)
(Look for MOTO G7 PLUS, you will see that the SKU No. 35 is selected, thanks to it's product property called `DefaultSKU`)

#### Screenshots or example usage:
For i.e:
By passing "SPECIFICATION" the SKU No. 35 was automatically selected in the Gallery

![item](https://user-images.githubusercontent.com/50715158/128880853-a9e5dd43-9f25-41de-95cc-f74b153a6299.png)

#### Describe alternatives you've considered, if any.

We know that the Admin SKU Position exists, but as of today, this is not working (Confirmed by Catalog team)

#### Related to / Depends on

[Search result PR](https://github.com/vtex-apps/search-result/pull/536)

#### How does this PR make you feel? 

![](https://media.giphy.com/media/5UB7rGI71YHBsuRQdi/giphy.gif)
